schema/categories_schema.cue

```diff
@@ -0,0 +1,41 @@
+package shopify
+
+#categoryId: =~"^gid://shopify/Taxonomy/Category/\\w+(-\\d+)*$"
```

Comment:
Nit: `\w+` will include underscore and digits. If we want to be stricter we can go with: `[a-z]{2}`. Also use for `prefix` in `verticals`.